### PR TITLE
Update ReadMe with instructions

### PR DESCRIPTION
**Pull request by kanchan-sannidhanam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
-# ccx-hackathon
\ No newline at end of file
+# ccx-hackathon
```
Comment (sfyz) [inline README.md]:
```suggestion
# Learning Map Generator Setup Instructions```

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+
+#Install Hugo:
```
Comment (sfyz) [inline README.md]:
```suggestion
## Install Hugo
```

File: README.md [hunk 3]
```diff
@@ -1 +1,36 @@
...
+macOs: $ brew install hugo
```
Comment (sfyz) [inline README.md]:
```suggestion
macOS: $ brew install hugo
```

File: README.md [hunk 4]
```diff
@@ -1 +1,36 @@
...
+To install on other platforms visit: https://gohugo.io/
+
+#Check version:
```
Comment (sfyz) [inline README.md]:
```suggestion
## Check Hugo Version
```

File: README.md [hunk 5]
```diff
@@ -1 +1,36 @@
...
+$ hugo version
+
+#Clone Learning Map Generator
```
Comment (sfyz) [inline README.md]:
```suggestion
## Clone Learning Map Generator
```

File: README.md [hunk 6]
```diff
@@ -1 +1,36 @@
...
+$ git clone https://github.com/tyrin/learning-map-generator.git
+
+#Build the site locally. This step will create a public folder where all the static content & images will copied to.
```
Comment (sfyz) [inline README.md]:
```suggestion
## Build Site Locally

 This step creates a public folder where all the static content & images are copied to. 
```

File: README.md [hunk 7]
```diff
@@ -1 +1,36 @@
...
+To install on other platforms visit: https://gohugo.io/
+
+#Check version:
+$ hugo version
+
+#Clone Learning Map Generator
+$ git clone https://github.com/tyrin/learning-map-generator.git
+
+#Build the site locally. This step will create a public folder where all the static content & images will copied to. 
+#The site content are loaded from the public folder
```
Comment (sfyz) [inline README.md]:
```suggestion
The site content are loaded from the public folder. To create the public folder run hugo without any arguments:
```

File: README.md [hunk 8]
```diff
@@ -1 +1,36 @@
...
+hugo
```
Comment (sfyz) [inline README.md]:
```suggestion
`$ hugo`
```

File: README.md [hunk 9]
```diff
@@ -1 +1,36 @@
...
+
+#Locally load the static site. This helps you visualize & validate what your site looks like
```
Comment (sfyz) [inline README.md]:
```suggestion
# Locally Load Static Site

This helps you visualize & validate what your site looks like.
```

File: README.md [hunk 10]
```diff
@@ -1 +1,36 @@
...
+$ hugo server
```
Comment (sfyz) [inline README.md]:
```suggestion
`$ hugo server`
```

File: README.md [hunk 11]
```diff
@@ -1 +1,36 @@
...
+open browser and go to http://localhost:1313
```
Comment (sfyz) [inline README.md]:
```suggestion
Open browser and go to `http://localhost:1313`.
```

File: README.md [hunk 12]
```diff
@@ -1 +1,36 @@
...
+press ctrl+c to stop the server
```
Comment (sfyz) [inline README.md]:
```suggestion
Press ctrl+c to stop the server.
```

File: README.md [hunk 13]
```diff
@@ -1 +1,36 @@
...
+
+#Deploy using Netlify
```
Comment (sfyz) [inline README.md]:
```suggestion
## Deploy using Netlify
```

File: README.md [hunk 14]
```diff
@@ -1 +1,36 @@
...
+1. go to https://app.netlify.com/
```
Comment (sfyz) [inline README.md]:
```suggestion
1. Go to https://app.netlify.com/
```

**Review by sfyz — APPROVED** [review]
Looking great. Just made some markdown header organizing suggestions...

File: README.md (hunk 1 above)
Comment (kanchan-sannidhanam) [reply README.md]: Done.

**Review by sfyz — APPROVED** [review]
👍

**Pull request merged**